sdk/src/main/java/io/opentelemetry/sdk/resources/EnvVarResource.java

```diff
@@ -29,12 +31,11 @@
  */
 @ThreadSafe
 public final class EnvVarResource {
```

Comment:
The name of the class seems to be incorrect now. At least `Var` part.

Reply:
Ah, now that it supports system properties also, got it (took me some time 😂)

Reply:
@iNikem Oh, indeed, I was meant to mention the rename as well.I suggest it being renamed to something like `EnvAutodetectResource` or `AutodetectResource`, and update its javadoc to mention that now we do env vars + system properties detection.

Reply:
Agree. @iNikem @trask @thisthat WDYT?

Reply:
I would prefer `EnvAutodetectResource` over `AutodetectResource` since we only look for a specific env var/sys property.

Reply:
Already renamed class name EnvVarResource to EnvAutodetectResource and update doc.